src/command_modules/azure-cli-storage/HISTORY.rst

```diff
@@ -2,6 +2,11 @@
 
 Release History
 ===============
+
+unreleased
+++++++++++
+* Remove --marker option from storage blob list, storage container list, and storage share list commands. #3745
```

Comment:
This is technically a breaking change.  Even if it didn't do anything before, any script that previously had it will now fail.